lobsterpy/featurize/core.py

```diff
@@ -620,57 +835,60 @@ def _calc_moment_features(
 
         try:
-            coxx_center = self._get_coxx_center(
-                coxx=coxx_dict[self.feature_type],
+            coxx_center = FeaturizeCOXX.get_coxx_center(
+                coxx=coxx_dict[feature_type],
                 energies=energies,
-                e_range=self.e_range,
+                e_range=e_range,
             )
             coxx_width = np.sqrt(
-                self._get_n_moment(
+                FeaturizeCOXX.get_n_moment(
                     n=2,
-                    coxx=coxx_dict[self.feature_type],
+                    coxx=coxx_dict[feature_type],
                     energies=energies,
-                    e_range=self.e_range,
+                    e_range=e_range,
                 )
             )
-            coxx_skew = self._get_n_moment(
+            coxx_skew = FeaturizeCOXX.get_n_moment(
                 n=3,
-                coxx=coxx_dict[self.feature_type],
+                coxx=coxx_dict[feature_type],
                 energies=energies,
-                e_range=self.e_range,
-            ) / self._get_n_moment(
+                e_range=e_range,
+            ) / FeaturizeCOXX.get_n_moment(
                 n=2,
-                coxx=coxx_dict[self.feature_type],
+                coxx=coxx_dict[feature_type],
                 energies=energies,
-                e_range=self.e_range,
+                e_range=e_range,
             ) ** (
                 3 / 2
             )
 
             coxx_kurt = (
-                self._get_n_moment(
+                FeaturizeCOXX.get_n_moment(
                     n=4,
-                    coxx=coxx_dict[self.feature_type],
+                    coxx=coxx_dict[feature_type],
                     energies=energies,
-                    e_range=self.e_range,
+                    e_range=e_range,
                 )
-                / self._get_n_moment(
+                / FeaturizeCOXX.get_n_moment(
                     n=2,
-                    coxx=coxx_dict[self.feature_type],
+                    coxx=coxx_dict[feature_type],
                     energies=energies,
-                    e_range=self.e_range,
+                    e_range=e_range,
                 )
                 ** 2
             )
+            coxx_edge = FeaturizeCOXX.get_cohp_edge(
+                coxx=coxx_dict[feature_type], energies=energies, e_range=e_range
+            )
         except KeyError:
             raise ValueError(
-                "Please recheck fp_type requested argument.Possible options are bonding/antibonding/overall"
+                "Please recheck feature type requested argument. Possible options are bonding/antibonding/overall"
             )
 
-        return coxx_center, coxx_width, coxx_skew, coxx_kurt
+        return coxx_center, coxx_width, coxx_skew, coxx_kurt, coxx_edge
 
-    def _get_coxx_center(
-        self,
+    @staticmethod
+    def get_coxx_center(
         coxx: npt.NDArray[np.floating],
         energies: npt.NDArray[np.floating],
```

Comment:
what type is this?

Reply:
Numpy array , will change it to usual notation

Reply:
Npt is confusing